ui/src/app/job-list/job-list.component.css

```diff
@@ -1,5 +1,8 @@
 .table-container {
   position: relative;
+  overflow: scroll;
+  width: 100%;
+  height: 100%;
```

Comment:
Does this change add vertical scrolling to the job list as well?

Reply:
Nope no-op change, removed.